app/modules/assets/models.py

```diff
@@ -259,6 +259,11 @@ def original_changed(self, image_object):
         # Save the new image
         image_object.save(symlink.resolve())
         self.reset_derived_images()
+        log.info(f'Rerunning detection, deleting annotations {self.annotations}')
+        for annotation in self.annotations:
+            annotation.delete()
```

Comment:
wondering if we should have an `audit_log` on each annot deletion?   for that inevitable "where the heck did this annotation go??" moment.   makes me also wonder, do we want to log on the _asset_ that detection was rerun?  (put another way: what is _too much_ logging? haha)

Reply:
I think these are valid questions but I would say outside of the scope of this PR, which has grown too much already